### PR TITLE
Bump rubocop version

### PR DESCRIPTION
**Pull request by edwardkerry**
This will make it possible to use govuk-lint on 2.6 projects.

### CODE REVIEW

**Comment by tijmenb** [comment]
Depending on the changes in rubocop, this version could trigger a lot of new rubocop issues, which means this version might not be quickly mergeable on projects that use it.

Could you run this gem version in a project that is fully linted, like https://github.com/alphagov/finder-frontend? If it raises a lot of new issues, we could add them to the rubocop.yml in this PR.

**Comment by edwardkerry** [comment]
Hi @tijmenb I've run this locally against `finder-frontend` and no new issues were reported. Checking the Rubocop changelog, although it's moving up a few versions most of the changes are bug-fixes with only a few new rules introduced. https://github.com/rubocop-hq/rubocop/blob/master/CHANGELOG.md

**Review by tijmenb — APPROVED** [review]
Awesome, thanks for checking.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Unreleased
+* Bump Rubocop to version 0.64 to fix Ruby 2.6.1 issue
+* Bump Ruby to 2.6.1
```
Comment (cbaines) [inline CHANGELOG.md:3]: This entry is a bit unnecessary, as the `.ruby-version` is just an internal thing.

**Pull request merged**